### ISSUE TITLE
Crash on game startup with Sodium 0.4.11

### ISSUE THREAD

**Issue by NothingSpecific** [StartsMercury/simply-no-shading#110]
When paired with Sodium 0.4.11+build.26, the game crashes on startup. Crash log: https://mclo.gs/21qM6Wq
This issue has already been posted on Sodium's bug tracker, but they said it needed to be fixed here instead.

https://github.com/CaffeineMC/sodium-fabric/issues/1723

**Comment by StartsMercury** [comment]
I heard that Sodium 0.4.11 was (intentionally) removed for downloads, especially from Modrinth. Simply No Shading wouldn't be able or would support Sodium 0.4.11 (yet) due to this. Also, [sodium-fabric#1746](https://github.com/CaffeineMC/sodium-fabric/issues/1746) stated that 0.4.11 was removed due to some (or maybe more issues), hence why it won't be supported (yet).

#### TLDR

Sodium 0.4.11 is not available now (or yet), try using 0.4.10 and come back if you found more issues.
